Extension/src/common.ts

```diff
@@ -538,7 +541,7 @@ export function createDirIfNotExistsSync(filePath: string | undefined): void {
     }
     const dirPath: string = path.dirname(filePath);
     if (!checkDirectoryExistsSync(dirPath)) {
-        fs.mkdirSync(dirPath);
+        fs.mkdirSync(dirPath, { recursive: true });
```

Comment:
Is there a bug associated with this change?

Reply:
No, but you get the same symptoms as #10231 (terminal hangs).